content/analyst/general/how-to-setup-sso.md

```diff
@@ -0,0 +1,55 @@
+---
+lastUpdated: "08/23/2023"
+title: How to Setup SSO
+description: This guide helps go through the process of setting up SSO for your account.
+---
+
+**To begin, navigate to the Account SSO Configuration page by clicking your email address in the top right corner.**
+
+ _**Note** that if you are not an assigned Account Manager, you will not see the Account SSO Configuration section. Another manager on the account can give you managerial permissions to access this setting. Please reach out to us if you are unsure of who the current account manager is._
+
+![](media/general_how_to_setup_sso_5.png)
```

Comment:
wondering if you want to get this png with a fake user/domain so you don't get spammed 😄